content/papers/2022-coverage-maximizing.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: papers
+title:  "Coverage-Maximizing Solar-Powered Autonomous Surface Vehicle Control for Persistent Gulf Stream Observation"
+date: 2022-06-10
+image: /images/default_paper.png
+venue: "IEEE ACC 2022"
+authors: 
+    - kavinmgovindarajan
+    - Ben Haydon
```

Comment:
just an FYI, I added the rest of the authors too. If the person's page exists, it will be cross linked, otherwise it will just print whatever you type here.